src/algebra/group_ring_action.lean

```diff
@@ -65,6 +65,14 @@ def mul_semiring_action.to_ring_equiv [mul_semiring_action G R] (x : G) : R ≃+
 { .. distrib_mul_action.to_add_equiv R x,
   .. mul_semiring_action.to_ring_hom G R x }
 
+/-- Compose a `mul_semiring_action` with a `monoid_hom`, with action `f r' • m`.
+See note [reducible non-instances]. -/
+@[reducible] def mul_semiring_action.comp_hom (g : K →* M) [mul_semiring_action M R] :
```

Comment:
Don't forget to change K for N here too

Reply:
Also, the docstring says `f` but you called the function `g`! Can you check what the definitions you're building upon do and emulate those?